Fail to build on raspberry pi

##### Bug report summary
Failed to build with either one-line or manual installation on raspberry pi 3a+ and zero w.
##### OS / Environment
Raspbian July 2019
##### Netdata version (ouput of `netdata -V`)
v1.17.1 (latest)
##### Component Name
x
##### Steps To Reproduce
Run one-line installation.
##### Expected behavior
x

##### Logs
```
  ...
  CC       daemon/common.o
  CC       daemon/daemon.o
collectors/slabinfo.plugin/slabinfo.c: In function 'read_file_slabinfo':
collectors/slabinfo.plugin/slabinfo.c:18:24: error: 'PAGE_SIZE' undeclared (first use in this function); did you mean 'PAGE_LOGS'?
 #define SLAB_PAGE_SIZE PAGE_SIZE
                        ^~~~~~~~~
collectors/slabinfo.plugin/slabinfo.c:216:51: note: in expansion of macro 'SLAB_PAGE_SIZE'
         uint32_t memperslab = s->pages_per_slab * SLAB_PAGE_SIZE;
                                                   ^~~~~~~~~~~~~~
collectors/slabinfo.plugin/slabinfo.c:18:24: note: each undeclared identifier is reported only once for each function it appears in
 #define SLAB_PAGE_SIZE PAGE_SIZE
                        ^~~~~~~~~
collectors/slabinfo.plugin/slabinfo.c:216:51: note: in expansion of macro 'SLAB_PAGE_SIZE'
         uint32_t memperslab = s->pages_per_slab * SLAB_PAGE_SIZE;
                                                   ^~~~~~~~~~~~~~
make[2]: *** [Makefile:3109: collectors/slabinfo.plugin/slabinfo.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[2]: Leaving directory '/tmp/netdata-kickstart-w4IuUQ/netdata-v1.17.1-66-g6dd4a11e'
make[1]: *** [Makefile:3231: all-recursive] Error 1
make[1]: Leaving directory '/tmp/netdata-kickstart-w4IuUQ/netdata-v1.17.1-66-g6dd4a11e'
make: *** [Makefile:1836: all] Error 2
 FAILED   

 FAILED   

 ABORTED  netdata-installer.sh exited with error
```

--- a/collectors/slabinfo.plugin/slabinfo.c
+++ b/collectors/slabinfo.plugin/slabinfo.c
@@ -10,10 +10,6 @@
 #define CHART_FAMILY "slab"
 #define CHART_PRIO 3000
 
-// As we're talking about kernel-pagesize, there's no hugepage.
-// So it's reliable to use it as an arch constant
-#define SLAB_PAGE_SIZE PAGE_SIZE
-
 // #define slabdebug(...) if (debug) { fprintf(stderr, __VA_ARGS__); }
 #define slabdebug(args...) if (debug) { \
     fprintf(stderr, "slabinfo.plugin DEBUG (%04d@%-10.10s:%-15.15s)::", __LINE__, __FILE__, __FUNCTION__); \

--- a/collectors/slabinfo.plugin/slabinfo.c
+++ b/collectors/slabinfo.plugin/slabinfo.c
@@ -3,9 +3,6 @@
 #include "libnetdata/libnetdata.h"
 #include "daemon/common.h"
 
-// For PAGE_SIZE
-#include <sys/user.h>
-
 #define PLUGIN_SLABINFO_NAME "slabinfo.plugin"
 #define PLUGIN_SLABINFO_PROCFILE "/proc/slabinfo"
 
@@ -170,6 +167,12 @@ struct slabinfo *read_file_slabinfo() {
     slabdebug("-> Reading procfile %s", PLUGIN_SLABINFO_PROCFILE);
 
     static procfile *ff = NULL;
+	static long slab_pagesize = 0;
+
+	if (unlikely(!slab_pagesize)) {
+		slab_pagesize = sysconf(_SC_PAGESIZE);
+		slabdebug("   Discovered pagesize: %ld", slab_pagesize);
+	}
 
     if(unlikely(!ff)) {
         ff = procfile_reopen(ff, PLUGIN_SLABINFO_PROCFILE, " ,:" , PROCFILE_FLAG_DEFAULT);
@@ -213,7 +216,7 @@ struct slabinfo *read_file_slabinfo() {
         s->data_num_slabs    = str2uint64_t(procfile_lineword(ff, l, 12));
         s->data_shared_avail = str2uint64_t(procfile_lineword(ff, l, 13));
 
-        uint32_t memperslab = s->pages_per_slab * SLAB_PAGE_SIZE;
+        uint32_t memperslab = s->pages_per_slab * slab_pagesize;
         // Internal fragmentation: loss per slab, due to objects not being a multiple of pagesize
         //uint32_t lossperslab = memperslab - s->obj_per_slab * s->obj_size;
 

